### PR TITLE
NxMeter - RSC-991

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-991

### CODE REVIEW

File: gallery/src/components/NxMeter/NxMeterPage.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                Meter Attributes
+              </NxTextLink>
+              , excluding <NxCode>min</NxCode>, <NxCode>low</NxCode>, <NxCode>high</NxCode>,
+              and <NxCode>optimum</NxCode>
+            </NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell />
+            <NxTable.Cell>
+              NxMeter supports any html attribute that are normally supported by HTML
+              <NxCode>&lt;meter&gt;</NxCode> elements, with the exception of <NxCode>min</NxCode>,
```
Comment (cindyko226) [inline gallery/src/components/NxMeter/NxMeterPage.tsx:83]:
```suggestion
              NxMeter supports any html attribute that are normally supported by HTML{' '}
              <NxCode>&lt;meter&gt;</NxCode> elements, with the exception of <NxCode>min</NxCode>,{' '}
```

File: gallery/src/components/NxMeter/NxMeterPage.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          </NxTable.Row>
+        </NxTable.Body>
+      </NxTable>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="NxMeter"
+                        id="nx-meter-example"
+                        liveExample={NxMeterExample}
+                        codeExamples={nxMeterExampleCode}>
+      Examples of <NxCode>NxMeter</NxCode> showing 0% measurements, 100% meaurements, and in-between measurements.
```
Comment (cindyko226) [inline gallery/src/components/NxMeter/NxMeterPage.tsx:96]:
```suggestion
      Examples of <NxCode>NxMeter</NxCode> showing 0% measurements, 100% measurements, and in-between measurements.
```

Comment (rpokorny) [reply gallery/src/components/NxMeter/NxMeterPage.tsx]: 61c8fd6f7

File: gallery/src/components/NxMeter/NxMeterPage.tsx (hunk 1 above)
Comment (rpokorny) [reply gallery/src/components/NxMeter/NxMeterPage.tsx]: 61c8fd6f7

**Review by cindyko226 — APPROVED** [review]

**Pull request merged**